src/login/Login.jsx

```diff
@@ -0,0 +1,133 @@
+import React, { useState } from "react";
+import "./Login.css";
+import { Redirect } from "react-router-dom";
+import "../myspace/MySpace";
+
+
+export default function Login() {
+    const [username, setUsername] = useState(true);
+    const [password, setPassword] = useState(true);
+    const [responseMessage, setResponseMessage] = useState(null);
+    const [accessToken, setAccessToken] = useState(null);
+    const [accessExpiry, setAccessExpiry] = useState(null);
+    const [isAuthenticated, setIsAuthenticated] = useState(false);
+
+    const handleSubmit = async e => {
+        e.preventDefault();
+        let userInput = {}
+
+        new FormData(e.target).forEach((value, key) => {
+            if (key === "username")
+                userInput[key] = value;
+            else if (key === "password")
+                userInput[key] = value;
+
+        });
+
+
+        const payload = {
+            username: userInput["username"],
+            password: userInput["password"],
+        };
+        const requestLogin = {
+            method: "POST",
+            headers: {
+                "Accept": "application/json",
+                "Content-Type": "application/json",
+            },
+            body: JSON.stringify(payload)
+
+        };
+        let data = "";
+        let resStatus = 0;
+        fetch("http://127.0.0.1:5000/login", requestLogin)
+            .then(async response => {
+                resStatus = response.status
+                data = await response.json();
```

Comment:
This can be refactored. A switch is not required. I can catchup with you to help refactor.

Reply:
yep, let's catch up tonight same time?

Reply:
I will be available at 7pm IST today. would that work for you?